src/test/java/org/jenkinsci/plugins/workflow/cps/view/InterpolatedSecretsActionTest.java

```diff
@@ -0,0 +1,47 @@
+package org.jenkinsci.plugins.workflow.cps.view;
```

Comment:
You should add a license header to this file, the same one you used for `summary.jelly` is fine.

Reply:
license added in 41e50e673469336413362acaeb5acbc896057d36